test/parallel/test-webcrypto-keygen.js

```diff
@@ -646,3 +656,8 @@ const vectors = {
 assert.throws(() => new CryptoKey(), {
   code: 'ERR_OPERATION_FAILED'
 });
+
+{
+  const buffer = Buffer.from('Hello World');
+  assert(!KeyObject.isCryptoKey(buffer));
```

Comment:
likewise here... a test asserting that `KeyObject.icCryptoKey(aKeyObject)` is `false`